upgrading/upgrade-operator.adoc

```diff
@@ -11,6 +11,11 @@ Upgrades through the {rh-rhacs-first} Operator are performed automatically or ma
 
 If you installed {product-title-short} using the Operator and selected *Automatic* in the *Update approval* field, {product-title-short} is automatically updated when a new software version is released. If you selected *Manual*, you must approve subsequent Operator updates by using Operator Lifecycle Manager (OLM). For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update].
 
+[NOTE]
+====
+You cannot use OLM, even with manual approval, to upgrade from {product-title-short} version 3.74 and earlier to {product-title-short} 4.0 because of database changes introduced in {product-title-short} 4.0. For that upgrade procedure, see "Upgrading from {product-title-short} 3.74 and earlier to {product-title-short 4.0 and later}" in the "Additional resources" section.
```

Comment:
I'd change "You cannot" to "it's not possible" because it is indeed technically not possible.

Something in the spirit of:
```suggestion
It is not possible, even with manual approval, to upgrade from {product-title-short} version 3.74 and earlier to {product-title-short} 4.0 via the standard OLM upgrade process. The custom upgrade procedure requires a manual intervention in order to give the opportunity for the system administrator to take a database backup before database changes introduced in {product-title-short} 4.0 are applied. For that upgrade procedure, see "Upgrading from {product-title-short} 3.74 and earlier to {product-title-short 4.0 and later}" in the "Additional resources" section.
```